### PR TITLE
Add react-native-http-bridge-refurbished

### PR DESCRIPTION
**Pull request by Alwinator**
## 📝 Why & how
Added react-native-http-bridge-refurbished library

## ✅ Checklist

- [X] Added library to **`react-native-libraries.json`**

### CODE REVIEW

File: react-native-libraries.json [hunk 1]
```diff
@@ -9823,5 +9823,18 @@
     "examples": ["https://github.com/Malaa-tech/react-native-simple-keypad/tree/main/example"],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/Alwinator/react-native-http-bridge-refurbished",
+    "npmPkg": "react-native-http-bridge-refurbished",
+    "examples": [
+      "https://github.com/Alwinator/react-native-http-bridge-refurbished#example"
+    ],
```
Comment (Simek) [inline react-native-libraries.json:9832]:
```suggestion
```

We consider apps as examples, so remove the Readme link. User will land on the GitHub page by licking the package name, so they will see the Readme content anyway.

File: react-native-libraries.json [hunk 2]
```diff
@@ -9823,5 +9823,18 @@
...
+    "ios": true,
+    "android": true,
+    "web": false,
+    "expo": false,
```
Comment (Simek) [inline react-native-libraries.json:9836]:
```suggestion
```

Fals flags can be skipped.

File: react-native-libraries.json [hunk 3]
```diff
@@ -9823,5 +9823,18 @@
...
+    "windows": true,
+    "macos": true
```
Comment (Simek) [inline react-native-libraries.json:9838]: Due to compatibility issues in the past, to accept the out-of-three platforms support, we require an example app which runs using `react-native-macos` fork and/or `react-native-windows` library based on selected platform.

**Review by Simek — COMMENTED** [review]
Hello @Alwinator, thank you for adding new library to the directory! 🙏 

Left a few, small suggestions. Unfortunately ATM there is a blocker in form of out-of-tree platforms support, please check the comment for the details.

With suggestions in, and additional platform out (for a while), we can ship this addition, and the desktop platform can be added via data update in the future, along side example app.

**Comment by Alwinator** [comment]
@Simek Changes done!

**Review by Simek — APPROVED** [review]
Thank you for the updates! 👍

**Pull request merged**